flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -4195,4 +4195,53 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "f55=f57",
       "true")
   }
+
+  //-------------------------------------------------------------------
+  // JSON functions
+  //-------------------------------------------------------------------
+  @Test
+  def testJsonQuery(): Unit = {
```

Comment:
Let's move json tests to a new test class, e.g. `JsonFunctionsTest`. `ScalarFunctionsTest` is too large now.